packages/render-fidelity-tools/test/config.json

```diff
@@ -253,7 +260,8 @@
         "radius": 0.5
       },
       "exclude": [
-        "dspbr-pt"
+        "dspbr-pt",
+        "stellar"
```

Comment:
This is one of the most important baseline cases; is there some reason stellar can't handle it?

Reply:
Our glTF loader doesn't support spec-gloss

Reply:
Oh sorry, I totally thought this was the metalRoughSpheresHDR. Yeah, this is fine to ignore.

Reply:
Added metalRoughSpheresHDR